alpha/external-web/app/routes/claim.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * This file defines all routes for the claim page.
+ */
+var router = require('../routes')
+var eligibilityFlag = require('../services/eligibility-flag')
+
+/**
+ * Renders the claim page.
+ *
+ * Example call: http://localhost:3000/claim
+ */
+router.get('/claim', function (request, response) {
+  console.log('GET /claim')
+  response.render('claim')
+})
+
+/**
+ * Ensure the claim form has been completed and redirect the user based on their response.
+ *
+ * Redirect the user to either:
+ *  - about-you     :: If their eligibility information has changed.
+ *  - claim-details :: If their eligibility information has not changed.
+ *
+ * Example call: http://localhost:3000/claim
+ */
+router.post('/claim', function (request, response) {
```

Comment:
a possible example of meaningful useful comment, tho my preference would be to have a test that explicitly covers this logic, documenting it that way